corehq/apps/domain/management/commands/bootstrap_app.py

```diff
@@ -0,0 +1,53 @@
+from django.core.management.base import LabelCommand, CommandError
+from django.conf import settings
+
+from corehq import Domain
+from corehq.apps.app_manager.const import APP_V2
+from corehq.apps.app_manager.models import Application, Module
+from corehq.util.view_utils import reverse
+
+from couchdbkit import ResourceNotFound
+
+
+class Command(LabelCommand):
+    help = "Bootstrap an Application with two modules in a given domain."
+    args = "<domain> <app-name>"
+    label = ""
+
+    def handle(self, *args, **options):
+
+        if len(args) != 2:
+            raise CommandError('Usage: manage.py bootstrap_app <domain> <app-name>')
+        domain_name, app_name = args
```

Comment:
Ideally, one would wish to pass an App-config and let this command create the app. But, that will be too much effort for now, this is a good starting point, I think.